### PR TITLE
Embeddings: improve wording on site admin page

### PR DESCRIPTION
**Pull request by jtibshirani**
This improves some wording around embedding jobs.
* For the sidebar, "Embeddings" -> "Embeddings jobs" and "Configuration" -> "Embeddings policies".
* Improved some page titles

The naming is now less consistent with the Code graph group, but that seems fine.

## Test plan

Manually inspected changes.

Screenshot 2023-06-07 at 4 52 53 PM

### CODE REVIEW

**Comment by jtibshirani** [comment]
Co-authored by Cody :) 

Screenshot 2023-06-07 at 4 41 54 PM

**Review by toolmantim — COMMENTED** [review]
Looking good!

Could we change the labels to use "Embeddings" instead of "Embedding"?

We probably need some language clarification somewhere, and @ryphil correct me if I'm wrong, but we'll have an "Embeddings API" not an "Embedding API" — I don't think we're planning to say "do an embedding" but more "generate embeddings". So it'll mostly be "Embeddings" everywhere.

**Comment by jtibshirani** [comment]
Oops, I standardized on "embedding jobs" and "embedding policies" everywhere in this PR :) It matches "indexing job" or "indexing policy". It makes sense because it's very common to use embed as a verb, like "embed this repository" or "embed this text". No strong opinions here though, happy to switch to whatever we want.

**Comment by toolmantim** [comment]
Yeah, I had the same conundrum on a bunch of design work! 😅 Once you have double plurals. Let's standardize on "Embeddings" for now.

**Comment by stefanhengl** [comment]
@toolmantim WDYT Should we change the page title to match the label in the sidebar?

Screenshot 2023-06-08 at 10 00 22

**Comment by toolmantim** [comment]
@stefanhengl yes! 💯😍

**Comment by jtibshirani** [comment]
I pushed an update to use plural "embeddings" everywhere, and make the page headers match the sidebar in terms of capitalization + wording.

I'd like to tackle @stefanhengl's suggestion in a follow-up, because it requires bigger code changes to do properly. To match other headings, we should style it with an icon and gray separator:

Screenshot 2023-06-08 at 9 13 29 AM

File: client/web/src/enterprise/site-admin/cody/SiteAdminCodyPage.tsx [hunk 1]
```diff
@@ -125,8 +125,8 @@ export const SiteAdminCodyPage: FC<SiteAdminCodyPageProps> = ({ telemetryService
 
     return (
         <>
-            <PageTitle title="Cody" />
-            <PageHeader path={[{ text: 'Cody' }]} className="mb-3" headingElement="h2" />
+            <PageTitle title="  Embeddings jobs" />
```
Comment (camdencheek) [inline client/web/src/enterprise/site-admin/cody/SiteAdminCodyPage.tsx:128]: Extra spaces? 

```suggestion
            <PageTitle title="Embeddings jobs" />
```

**Review by camdencheek — APPROVED** [review]

**Comment by jtibshirani** [comment]
Merging even though buildkite is red, since everything passes except "E2E tests for the Cody VS Code extension" are timing out. These are known flaky tests that I've flagged to the Cody editor team.

**Review by toolmantim — APPROVED** [review]
😍

**Pull request merged**